projects/components/src/button/button.component.scss

```diff
@@ -23,6 +23,7 @@
 
 .button-container {
   display: flex;
+  height: inherit;
```

Comment:
Should it inherit the height?

Reply:
The buttons in the latest mocks have a height of 36px. We are using 32px in several places and that is what the medium button is hard coded to. 

I was too worried about the consequences of changing the button height everywhere to make the leap of setting 36px as the default, so I did this to allow overriding. Though, we should do a sweep and change them all at some point.

Reply:
Hmm so what are the desired sizes?
In the mocks, I see:
```
X-small: 24px
Small: 32px
Default: 36px
Large: 40px
```
In code, I see:
```
X-small: 18px
Small: 24px
Medium: 32px
Large: 40px
```

Rather than change medium, which I agree is a bit risky, can we introduce a Default size of 36px and deprecate medium?

We'd still need to resolve x small and small, but it would avoid having to override anything which we generally don't want to allow.

Reply:
I think I'd prefer to just go through and fix these correctly all at once in a separate PR. It shouldn't be too bad to do.

Reply:
OK. Given that my lack-of-context opinion is that that we should just use the existing medium size until that's done, but leave it to your discretion.

Reply:
Yeah, thanks. Let's go with this override. I'll come back and fix the inconsistencies and remove the override separately.